### PR TITLE
Additional terraform example 

### PR DESCRIPTION
**Pull request by calbritt**

### CODE REVIEW

File: examples/env_with_dxe/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| kaleido_api_key | Kaleido API Key | string | - | yes |
+| kaleido_region | Can be '-ap' for Sydney, or '-eu' for Frankfurt. Defaults to US | string | `` | no |
+| provider | Blockchain Provider for the environment. | string | `pantheon` | no |
+| consensus | Consensus methods supported by pantheon. | list | `ibft` | no |
+| multi_region | Multi region env | string | `false` | no |
```
Comment (hfuss) [inline examples/env_with_dxe/README.md:15]: Let's make this default to `true` as this is what we suggest to all customers / envs now. See #44

**Review by hfuss — APPROVED** [review]
looks great, perhaps let's rename to `multi_region_with_multi_party` ?

File: examples/env_with_dxe/main.tf [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+/*
+Create DXE service
+*/
+resource "kaleido_service" "dxe" {
+  count = "${var.service_count}"
+  consortium_id = "${kaleido_consortium.consortium.id}"
+  environment_id = "${kaleido_environment.env.id}"
+  membership_id = "${element(kaleido_membership.member.*.id, count.index)}"
+  service_type = "dataexchange"
+  name = "IPFS ${count.index + 1}"
```
Comment (hfuss) [inline examples/env_with_dxe/main.tf:77]: Make sure to change to `DXE` instead

File: examples/env_with_dxe/main.tf [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  membership_id = "${element(kaleido_membership.member.*.id, count.index)}"
+  name = "IPFS ${count.index + 1}"
+  service_type = "ipfs"
+
+  depends_on = ["kaleido_node.kaleido"]
+}
+
+/*
+Create DXE service
+*/
```
Comment (hfuss) [inline examples/env_with_dxe/main.tf:70]: Perhaps let's leave a comment suggesting to contact support if you're interested in using DataExchange Enterprise for FireFly multi-party networks?

As well as some sort of plug for DataExchange Enterprise and FireFly in the README?

**Review by peterbroadhurst — APPROVED** [review]

**Pull request merged**